app/views/stats/_range.html.erb

```diff
@@ -93,6 +99,11 @@
                 <li>
                     <%= link_to "Questions Answered", stats_answers_path(format: 'csv', start: params[:start], end: params[:end]) %>
                 </li>
+                    <%= link_to "List of Tags", stats_tags_path(format: 'csv') %>
```

Comment:
```suggestion
                <li>
                    <%= link_to "List of Tags", stats_tags_path(format: 'csv') %>
```

Reply:
Opening tag needs to be opened here

Reply:
Also, @skilfullycurled there's an extra `<li>` at https://github.com/publiclab/plots2/pull/5459/files#diff-e8d970f814aa8ace0c4be3778174c22bR92. This is unrelated to your code, but I think we can remove that too. Thanks!